### ISSUE TITLE
after an update to CryptPad v1.23.0 (Xenomorph)

### ISSUE THREAD

**Issue by hapee** [xwiki-labs/cryptpad#218]
including npm install and bower install I can log in to Cryptpad and I see my existing documents and folders but I can not open them. I get a spinning wheel forever. Is there a way to find out what is going wrong and is there a way to fix it. Running on Ubuntu.

**Comment by hapee** [comment]
Very helpful so far, please can you point me at least to logfiles where I can find out if something goes wrong? this is not the only instance, I use both and both act similar.

**Comment by ansuz** [comment]
Hi,
there might be an error printed somewhere in your browser's console. Are you running with just the node process, or have you set it up behind a reverse proxy like Nginx?

**Comment by hapee** [comment]
I will have a look, it is behind the reverse proxy Nginx

**Comment by hapee** [comment]
I see a lot of (around 913) requests for 
https://crypt.drieendertig.nl/customize/src/less2/include/colortheme.less?ver=1.24.0
and spinning wheel with messages but no error and nothing further

**Comment by ansuz** [comment]
It helps immensely that you have a public domain I can visit.

I tried to register on your instance, and I saw one critical error:

> Error: Please update ChainPad

I believe `npm install && bower install` only add _missing_ dependencies. For our latest code, `npm update && bower update` should fetch the latest compatible versions of our libraries.

After updating, you'll need to change the version sent to clients for cache-busting (the 1.24.0 appended to the end of your URLs). This is served via `/api/config`, derived serverside from your package.json file.
